src/org/knotation/clj_api.clj

```diff
@@ -27,6 +28,7 @@
       (string/ends-with? path ".rdf") :rdfxml
       (string/ends-with? path ".owl") :rdfxml
       (string/ends-with? path ".edn") :edn
+      (string/ends-with? path ".xml") :drawio
```

Comment:
XML is too widely used to say it's just for drawio. I think we should drop this line and require the user to specify drawio format explicitly.